src/components/ThankYou.js

```diff
@@ -34,7 +34,7 @@ class ThankYou extends Component {
       <div className="flex flex-grow-1 bg-tan-white w-100 pt6 pb5">
         <div className="cf flex flex-column flex-row-l mw9 center w-100 ph3 mt5">
           <div className="fl w-100 w-40-l ph3">
-              <div className="relative z-1 h5 br3 bg-dark-gray w-100 shadow-3 pt2 overflow-scroll">
+              <div className="relative z-1 h5 br3 bg-dark-gray w-100 shadow-3 pt2 overflow-hidden">
```

Comment:
Can we remove `overflow-*` entirely? Do you think there's a reason we need to define overflow properties here?

Reply:
I don't think it was necessary, so I removed it completely